README.md

```diff
@@ -210,6 +210,93 @@ This app uses [RSpec](http://rspec.info/) for Rails tests.
 
     rake parallel:spec
 
+## Required environment variables for the
 
+* Prisons that are emailed via MessageLabs - Government so-called-"Secure" Intranet. You will need it, if you try locally to submit a booking request locally. Source: app/mailers/prison_mailer.rb
```

Comment:
Sorry, the "so-called" was me being sarcastic :-)

Perhaps a better explanation would be:

> These SMTP settings are used to send emails to prisons. Because these are on the Government Secure Intranet (GSI), they must be sent via MessageLabs.